ush/create_aqm_rc_file.py

```diff
@@ -19,74 +19,81 @@
     str_to_type,
 )
 
+
 def create_aqm_rc_file(cdate, run_dir, init_concentrations):
-    """ Creates an aqm.rc file in the specified run directory
+    """Creates an ``aqm.rc`` file in the specified run directory
 
     Args:
-        cdate: cycle date
-        run_dir: run directory
-        init_concentrations
+        cdate                (str): cycle date
```

Comment:
Should the c in cycle be capitalized?

```suggestion
        cdate                (str): Cycle date
```